CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 Changes
 =======
 
+# 2.17.0 / 2019-03-01
+
+* [FEATURE] Add support for SLES. See [#505][] [@gmmeyer][]
```

Comment:
the links to PRs need to be added to the bottom of the doc (fyi I'm using https://github.com/pcreux/pimpmychangelog to do that)